gordo_components/model/anomaly/diff.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+
+import os
+import json
+
+import numpy as np
+import pandas as pd
+
+from typing import Optional, Union
+from datetime import datetime, timedelta
+
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.base import BaseEstimator, TransformerMixin
+
+from gordo_components import serializer
+from gordo_components.model.base import GordoBase
+
+
+class DiffBasedAnomalyDetector(BaseEstimator, GordoBase):
+    def __init__(
+        self,
+        base_estimator: BaseEstimator = PCA(),
```

Comment:
Should we have a default here? Is it not better to demand a base estimator? And if we want a default, is PCA a sensible one?

Reply:
I think we definitely should have a default, as that's the Scikit-Learn way of things.. but agree, maybe PCA isn't the best. Maybe feedforward?